.github/workflows/code-coverage-test.yml

```diff
@@ -0,0 +1,418 @@
+name: Code Coverage Test
+
+env:
+  # TODO: we really need to define a list of supported versions (ideally it's no more than 2)
+  #       build is done on the lowest version and test on the highest with a "sanity test"
+  #       stage done on all versions in the list except the highest
+  EL8_BUILD_VERSION: 8.6
+  EL8_VERSION: 8.8
+  EL9_BUILD_VERSION: 9
+  EL9_VERSION: 9
+  LEAP15_VERSION: 15.5
+
+  JENKINS_URL: https://build.hpdd.intel.com/
+
+on:
+  pull_request:             # Remove after testing
```

Comment:
Running with this for now for testing.  We don't want to run this on every PR so the plan is to remove this line in the final push.